applications/tari_dan_wallet_cli/src/command/mod.rs

```diff
@@ -52,4 +53,6 @@ pub enum Command {
     WebRtc(WebRtcSubcommand),
     #[clap(subcommand, alias = "auth")]
     Auth(AuthSubcommand),
+    #[clap(subcommand, alias = "nfts")]
+    AccountNFT(AccountNFTSubcommand),
```

Comment:
convention for acronyms is Capital Case: https://rust-lang.github.io/api-guidelines/naming.html
```suggestion
    AccountNft(AccountNftSubcommand),
```